util/dvsim/Deploy.py

```diff
@@ -283,6 +310,18 @@ def get_timeout_mins(self):
         """Returns the timeout in minutes."""
         return 0
 
+    def get_runtimes(self, lines):
+        """Input the logfile in lines and extract the job_runtime."""
+        try:
+            self.job_runtime.time, self.job_runtime.unit = get_job_runtime(
+                lines, self.sim_cfg.tool)
+        except NotImplementedError as e:
+            #TODO: use dvsim default runtime.
+            log.debug("Using dvsim default runtime due to unsupported tool: " + self.sim_cfg.tool)
+        except SyntaxError as e:
+            #TODO: use dvsim default runtime.
+            log.debug("Parsing job runtime has syntax error with " + self.sim_cfg.tool)
```

Comment:
Documented here as enhancement: https://github.com/lowRISC/opentitan/issues/13903
This PR: https://github.com/lowRISC/opentitan/pull/13865 just merged, so should be a small patch here in the coming PR to compute the current job_runtime with the correct unit.